draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -917,20 +964,19 @@ presented at the same time, etc.
 It is important that a Verifier does not make its security policy
 decisions based on data that can be influenced by an attacker or that
 can be misinterpreted. For this reason, when deciding whether Key
-Binding is required or not, Verifiers MUST NOT take into account
+Binding is required or not, Verifiers MUST NOT take into account:
 
- * whether a Key Binding JWT is present or not, as an attacker can
-   remove the Key Binding JWT from any Presentation and present it to the
-   Verifier, or
+ * whether the Holder has provided an SD-JWT+KB or a bare SD-JWT, since otherwise a
+   attacker could strip the KB-JWT from an SD-JWT+KB and present the resulting
```

Comment:
```suggestion
 * whether the Holder has provided an SD-JWT+KB or a bare SD-JWT, since otherwise an
   attacker could strip the KB-JWT from an SD-JWT+KB and present the resulting
```